students/andrew_garcia/lesson_03/List_Lab.py

```diff
@@ -0,0 +1,110 @@
+'''
+Andrew Garcia
+List Lab
+6/15/19
+'''
+
+#!/usr/bin/env python3
+
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruit_list)
+
+
+# Sequence 1
+#print('\n' +'-------SEQUENCE 1-------')
+# add new fruit to list
+new_fruit = input("Add a fruit to the list: ")
+fruit_list.append(new_fruit.title())
+print(fruit_list)
+
+# Print a fruit based on index input
+fruit_number = int(input("Pick a number between 1 and " + str(len(fruit_list)) + ": "))
+fruit_index = fruit_number - 1
+print("You picked number {}, which means you picked {} \n".format(fruit_number, fruit_list[fruit_index]))
+
+# Add two new fruits to beginning of list
+new_fruit2 = input("Add a fruit to the list: ")
+fruit_list = [new_fruit2.title()] + fruit_list
+print(fruit_list, '\n')
+
+
+new_fruit3 = input("Add a fruit to the list: ")
+fruit_list.insert(0, new_fruit3.title())
+print(fruit_list)
+
+# Print fruits starting with P
+print('\n' + "The fruits in your list that start with P are:")
+for item in fruit_list:
+    if 'P' in item[0]:
```

Comment:
using `==` is more applicable since `in` tests for element inside of a collection and here you're testing character against character.